WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -127,6 +127,29 @@ $venue-map-wrapper-height: 400px;
 }
 
 .competitions-table {
+  font-size: 12px;
```

Comment:
I did, but bootstrap's default is 14px, which is too big for this (most of the time) long list.